attributes/server_web2.rb

```diff
@@ -11,3 +11,4 @@
 default['icinga2']['web2']['web_uri'] = '/icingaweb2'
 default['icinga2']['web2']['conf_dir'] = '/etc/icingaweb2'
 default['icinga2']['web2']['log_dir'] = '/var/log/icingaweb2'
+default['icinga2']['web2']['release'] = '1'
```

Comment:
Perhaps Debian doesn't care about the value, but if it doesn't get set to _something_, the cookbook doesn't compile (AFAICT)